### PR TITLE
Using references_added function for user's ref count

### PR DESCRIPTION
**Pull request by khyatisoneji**

### CODE REVIEW

**Comment by ragesoss** [comment]
@khyatisoneji please have a look at the test that failed.

File: spec/presenters/individual_statistics_presenter_spec.rb [hunk 1]
```diff
@@ -40,7 +37,7 @@
         expect(course2.revisions.count).to eq(1)
         expect(subject.individual_article_views).to eq(100)
         expect(subject.individual_character_count).to eq(100)
-        expect(subject.individual_references_count).to eq(5)
+        expect(subject.individual_references_count).to eq(22)
```
Comment (ragesoss) [inline spec/presenters/individual_statistics_presenter_spec.rb]: Why didn't this work the way it was before?

Comment (khyatisoneji) [reply spec/presenters/individual_statistics_presenter_spec.rb]: Here the example taken is for `newarticle`, so it'll return only current `ref_tags`, earlier I was just directly just subtracting, now I'm calling the function

So, I was made changes accordingly for `newarticle` revision example

Comment (ragesoss) [reply spec/presenters/individual_statistics_presenter_spec.rb]: Ah! That makes sense, thanks.

**Pull request merged**